tests/flow/test_union.py

```
@@ -171,4 +171,48 @@ def test09_union_write_read(self):
+               UNION
+
+               MATCH (p)
+               WITH p
+               RETURN 2 AS x"""
+
+        try:
+            self.graph.query(q)
+            # fail if query did not returned an error
+            self.env.assertFalse(True)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)